### DISCUSSION TITLE
Line Heights

### DISCUSSION THREAD

**Discussion by craig-willers** [six7/figma-tokens#115] [General]
Hey, Struggling with line-heights by %. ie, adding 120% rather than a point size. Bug or am I being thick? 

This is in the Line Heights setting (not the Typography row).

**Comment by six7** [comment]
Thanks for reporting! Confirmed there seems to be a bug when applying line heights independently directly on nodes.

Will get a fix online today.

**Reply by six7** [reply]
Version 51 just got released with a fix for this! 🙏
